i18n/states.php

```diff
@@ -1485,6 +1485,33 @@
 		'TZ30' => __( 'Simiyu', 'woocommerce' ),
 	),
 	'LK' => array(),
+	'RS' => array( // Serbia districts. Ref: http://www.statoids.com/urs.html
```

Comment:
Hello, 
Thanks for your help.

But note that now we follow the CLDR list, please update those based on this list: https://github.com/unicode-org/cldr/blob/release-37/common/subdivisions/en.xml#L4251-L4283

Reply:
Thank you for the link.